app/scripts/components/dashboardComponents/listItems.jsx

```diff
@@ -29,26 +30,35 @@ class ListItems extends React.Component {
 
     render() {
         const {
-            allItemsSelected, filesChecked, foldersChecked, isSafari, listItems,
-            loading, nextPage, projectRole, projectRoles, screenSize, tableBodyRenderKey,
-            totalItems, uploads
+            allItemsSelected, filesChecked, foldersChecked, isSafari,
+            loading, nextPage, projectRole, projectRoles, projects, screenSize,
+            tableBodyRenderKey, totalItems, uploads
         } = mainStore;
+        const { downloadedItems, listItems } = dashboardStore;
         const { agents } = agentStore;
         const { currentUser } = authStore;
-        let items = this.props.router && this.isListKind('Agents') ? agents.filter(a => a.audit.created_by.id === currentUser.id) : listItems;
-        let showBatchOps = this.isListKind('ProjectChildren') && !!(filesChecked.length || foldersChecked.length);
-        let showUploadButton = this.isListKind('ProjectChildren') && projectRole !== null && (projectRole !== Roles.project_viewer && projectRole !== Roles.file_downloader);
-        let showAddFolderButton = this.isListKind('ProjectChildren');
+        let items
+        if (this.isListKind('Projects')) {
+            items = projects;
+        } else if (this.isListKind('Agents')) {
+            items = agents.filter(a => a.audit.created_by.id === currentUser.id);
+        } else {
+            items = listItems;
+        }
+
+        let showBatchOps = this.isListKind('FoldersFiles') && !!(filesChecked.length || foldersChecked.length);
+        let showUploadButton = this.isListKind('FoldersFiles') && projectRole !== null && (projectRole !== Roles.project_viewer && projectRole !== Roles.file_downloader);
+        let showAddFolderButton = this.isListKind('FoldersFiles');
         let showAddProjectButton = this.isListKind('Projects');
         let showListItemsTable = items.length > 0;
         let showLoadMorebutton = !this.isListKind('Agents') && items.length < totalItems && totalItems > 25;
```

Comment:
Fix camel case. `showLoadMorebutton ` should be `showLoadMoreButton `

Reply:
fixed :)